bedrock/firefox/tests/test_helpers.py

```diff
@@ -0,0 +1,53 @@
+from django.test.utils import override_settings
+
+from mock import patch
+from nose.tools import assert_false, eq_
+
+from bedrock.mozorg.tests import TestCase
+from bedrock.firefox import helpers
+
+
+@override_settings(FIREFOX_OS_FEED_LOCALES=['xx'])
+@patch('bedrock.firefox.helpers.cache')
+@patch('bedrock.firefox.helpers.FirefoxOSFeedLink')
+class FirefoxOSFeedLinksTest(TestCase):
+    def test_no_feed_for_locale(self, FirefoxOSFeedLink, cache):
+        """
+        Should return None without checking cache or db.
+        """
+        eq_(helpers.firefox_os_feed_links('yy'), None)
```

Comment:
NIT: weird for me to see a test like `x == None` instead of `x is None`. I usually do `self.assertIsNone(x)` or `ok_(x is None)`. This is extremely minor and picky and you shouldn't change it, just wanted to get it out there in case I was missing something.